api4/apitestlib.go

```diff
@@ -420,6 +436,10 @@ func (th *TestHelper) CreateWebSocketSystemAdminClient() (*model.WebSocketClient
 	return model.NewWebSocketClient4(fmt.Sprintf("ws://localhost:%v", th.App.Srv().ListenAddr.Port), th.SystemAdminClient.AuthToken)
 }
 
+func (th *TestHelper) CreateWebSocketSystemManagerClient() (*model.WebSocketClient, *model.AppError) {
```

Comment:
Is this used?

Reply:
Nope. I just added it since it was added for System Admin. Is it better to remove or keep in case it needs to be used in the future?

Reply:
@sbishel